web/models.py

```
@@ -4,6 +4,7 @@
 import time
 import uuid
 from io import BytesIO
+from typing import ClassVar, List, Tuple
```

Ruff (0.8.2)

`typing.List` is deprecated, use `list` instead

(UP035)